repositories/dbmodels/db_decision_rule.go

```diff
@@ -3,30 +3,24 @@ package dbmodels
 import (
 	"github.com/checkmarble/marble-backend/models"
 	"github.com/checkmarble/marble-backend/models/ast"
-	"github.com/checkmarble/marble-backend/utils"
-
-	"github.com/jackc/pgx/v5/pgtype"
 )
 
 type DbDecisionRule struct {
-	Id             string             `db:"id"`
```

Comment:
Those tags are used with the method `SqlToListOfModels` - it's kind of convenient when reading from a single table, but its convenience really breaks down as soon as we do joins